### PR TITLE
Add io.gitlab.leesonwai.Tactics

### PR DESCRIPTION
**Pull request by leesonwai**
- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: io.gitlab.leesonwai.Tactics.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "app-id" : "io.gitlab.leesonwai.Tactics",
+  "runtime" : "org.gnome.Platform",
+  "runtime-version" : "43",
+  "sdk" : "org.gnome.Sdk",
+  "command" : "tactics",
+  "finish-args" : [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland"
```
Comment (hfiguiere) [inline io.gitlab.leesonwai.Tactics.json:10]: you don't need GL? since it's a Gtk4 app I'd expect to

**Comment by hfiguiere** [comment]
bot, build io.gitlab.leesonwai.Tactics

File: io.gitlab.leesonwai.Tactics.json (hunk 1 above)
Comment (leesonwai) [reply io.gitlab.leesonwai.Tactics.json:11]: Thanks for making me aware :smile:.

**Pull request closed without merging**